aws-shared-1/main.tf

```diff
@@ -0,0 +1,94 @@
+variable "env" { default = "shared" }
+variable "index" { default = "1" }
+variable "workers_com_subnet_1b" { default = "10.10.3.0/24" }
+variable "workers_com_subnet_1e" { default = "10.10.5.0/24" }
+variable "workers_org_subnet_1b" { default = "10.10.2.0/24" }
+variable "workers_org_subnet_1e" { default = "10.10.6.0/24" }
+
+provider "aws" {}
+
+resource "aws_vpc" "main" {
+  cidr_block = "10.10.0.0/16"
```

Comment:
Yes.  The intent here is to build out this infrastructure rooted at a different VPC.